bin/test-workflow/0_prep_env.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -o pipefail
+
+### PLATFORM DETAILS
+export CONJUR_OSS_HELM_INSTALLED="${CONJUR_OSS_HELM_INSTALLED:-true}"
+
+if [[ "$CONJUR_OSS_HELM_INSTALLED" == "true" ]]; then
+  TEST_PLATFORM="${TEST_PLATFORM:-kind}"
+else
+  TEST_PLATFORM="${TEST_PLATFORM:-gke}"
+  export UNIQUE_TEST_ID="$(uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10)"
+fi
+export TEST_PLATFORM
+
+if [[ "${TEST_PLATFORM}" == "oc" ]]; then
+  PLATFORM="openshift"
```

Comment:
After a little investigation, the only practical use of the long-form `PLATFORM` was when referencing directories of manifests (i.e. `bin/test-workflow/kubernetes` and `bin/test-workflow/openshift` which I assume is soon to come). Both were being used to run commands conditionally for Openshift.

I changed the long-form PLATFORM to MANIFEST_DIR, and TEST_PLATFORM to PLATFORM.
PLATFORM is used in conditionals.
MANIFEST_DIR is only used where the long-form "kubernetes" or "openshift" is needed.

Reply:
AH, thanks for the explanation. I was thinking that the distinction between these 2 would be something like this.
Either way, maybe we can add a comment just above this line, since I suspect other readers may wonder what the difference is here.